packages/mwp-core/src/util/cookieUtils.test.js

```diff
@@ -38,6 +44,21 @@ describe('parseMemberCookie', () => {
 	});
 });
 
+describe('parseBrowserIdCookie', () => {
+	it('returns a browser id', () => {
+		const requestState = { [BROWSER_ID_COOKIE]: 'id=abc-def-g1234' };
```

Comment:
`MEETUP_BROWSER_ID` actually includes double quotes around it - this test should be

```suggestion
		const requestState = { [BROWSER_ID_COOKIE]: '"id=abc-def-g1234"' };
```

(which I think will fail, but not sure)